Fronter compatibility

DrDudelsack opened the pull request:
Changes program to read configuration settings from configuration.txt generated by fronter, rather then typing into the console

DrDudelsack committed:
Fronter compatibility

Changes program to read configuration settings from configuration.txt generated by fronter, rather then typing into the console

DrDudelsack committed:
Update Readme.txt

DrDudelsack committed:
Update Readme.txt

DrDudelsack committed:
Update Readme.txt

DrDudelsack committed:
Update Readme.txt

IhateTrains commented on ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java:
```diff
@@ -1,707 +1,716 @@
+
+    public static String formatSaveName(String name) {// replaces /:*?"<>|- and space with _, while removing tabs and .rome
+        String quote = '"'+""; // " character, Java doesn't like isoated " characters
+        String tab = "    ";
```
This is actually a string of 4 spaces. I doubt you intended this?

DrDudelsack replied:
Indeed, lol, my IDE replaced hard tab characters with spaces. I'll try to upload a corrected version sometime tonight or tomorrow

DrDudelsack committed:
Title conversion mapping

Mappings of which I:R countries become which pre-existing CK II titles. Formatting is rank_I:R tag,rank_vanilla title. A tag's country rank (kingdom or empire) must match the vanilla title's rank. A small Rome in Imperator that's only a kingdom tier rank won't be converted as CK II's Roman Empire, for example. This allows titles to be converted without artificially upgrading or downgrading a country's rank

DrDudelsack committed:
Fixed Tabs, Added Ranks, Added Title conversion

Fixed corrupted tab characters, added the ability to create empire titles if I:R tag is big enough in CK II, and added the ability to convert certain I:R tags to vanilla title counterparts

DrDudelsack committed:
Fixed tabs

DrDudelsack committed:
Fixed tabs, allowed rank conversion

DrDudelsack committed:
Fixed tabs, added convertTitle(String,String,String)

Added convertTitle(String,String,String), which'll convert I:R tags to CK II vanilla titles

DrDudelsack committed:
Fixed tabs

IhateTrains commented on ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java:
```diff
@@ -1,707 +1,736 @@
+
+                    }
+                    aqq = 0;   
+                    if (impProv != "a") {
```
My Java knowledge is poor, but wouldn't

```suggestion
                    if (!"a".equals(impProv)) {
```

for string inequality checks be better?

DrDudelsack replied:
I'm not sure, but `impProv != "a"` works and doesn't cause any errors. If it works, it works

DrDudelsack committed:
Update Output.java

DrDudelsack committed:
Add files via upload

DrDudelsack committed:
Add files via upload

DrDudelsack committed:
Fixed Line endings

DrDudelsack committed:
Removed e_NUM : e_maghreb mapping

Removed Numidia formable (NUM) from being mapped to CK II's Maghreb title (e_maghreb), as CK II's Maghreb empire has little relation to Imperator:Rome's Numidia formable and because CK II's Maghreb Empire has no EU IV mapping in the CK II to EU IV converter

DrDudelsack committed:
Update Characters.java

DrDudelsack committed:
Update TempFiles.java

DrDudelsack committed:
Update Main.java

Fixed a bug regarding civil war tags by defaulting to regular tags in case there the title had no conversion, rather then the historical tag

DrDudelsack committed:
Update Processing.java

Fixed a bug regarding civil war tags by defaulting to regular tags in case there the title had no conversion, rather then the historical tag

DrDudelsack committed:
Changed tab characters

DrDudelsack committed:
Add files via upload

DrDudelsack committed:
Add files via upload

DrDudelsack committed:
Add files via upload

DrDudelsack committed:
Delete Release directory
